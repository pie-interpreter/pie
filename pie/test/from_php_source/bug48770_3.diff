004+ Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'C' does not have a method 'inexistent' in /home/sery0ga/work/php-src/Zend/tests/bug48770_3.php on line 22
004- Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'B' does not have a method 'inexistent' in %s on line %d