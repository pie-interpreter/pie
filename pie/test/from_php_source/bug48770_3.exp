%unicode|string%(27) "B::func2: This should work!"
%unicode|string%(27) "B::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'B' does not have a method 'inexistent' in %s on line %d