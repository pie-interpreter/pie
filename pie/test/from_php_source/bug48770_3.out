string(27) "B::func2: This should work!"
string(27) "B::func3: This should work!"

Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'C' does not have a method 'inexistent' in /home/sery0ga/work/php-src/Zend/tests/bug48770_3.php on line 22